CONTRIBUTING.md

```diff
@@ -81,7 +81,7 @@ Please contribute! Meshery documentation uses GitHub Pages to host the docs site
 1. Run site locally to preview changes.
    `make site`
```

Comment:
```suggestion
   `make docs-run`
```

While the `make site` command will still work, we _just_ updated this command to `make docs-run`.